CMIP6_source_id.json

```diff
@@ -838,44 +857,46 @@
             "label_extended":"CNRM-ESM2-1",
             "model_component":{
                 "aerosol":{
-                    "description":"TACTIC (v2.0)",
-                    "nominal_resolution":""
+                    "description":"TACTIC_v2",
+                    "nominal_resolution":"250 km"
                 },
                 "atmos":{
-                    "description":"Arpege 6.2 (Tl127; 91 levels; top level 78.4 km)",
-                    "nominal_resolution":""
+                    "description":"Arpege 6.3 (Tl127, 24572 x 1; XXX x XXX longitude/latitude; 91 levels; top level 78.4 km)",
```

Comment:
The entry 24572 x 1; XXX x XXX longitude/latitude; needs to be updated to a single correct value